src/utils/coana.mts

```diff
@@ -15,10 +15,20 @@ export async function spawnCoana(
   options?: SpawnOptions | undefined,
   extra?: SpawnExtra | undefined,
 ): Promise<CResult<unknown>> {
-  const { env: spawnEnv } = { __proto__: null, ...options } as SpawnOptions
+  const {
+    env: spawnEnv,
+    spinner,
+    stdio,
+  } = { __proto__: null, ...options } as SpawnOptions
   const orgSlugCResult = await getDefaultOrgSlug()
   const SOCKET_CLI_API_TOKEN = getDefaultToken()
   const SOCKET_ORG_SLUG = orgSlugCResult.ok ? orgSlugCResult.data : undefined
+
+  // Stop spinner before streaming output if stdio is 'inherit'
+  if (stdio === 'inherit' && spinner) {
+    spinner.stop()
+  }
```

Comment:
@mtorp ☝️ Nice, good catch!

There's an option to pass the spinner into `spawn` to have it handle that... but it will restore the spinner after I think. No worries. I'll noodle in the code to adjust in either spots as needed.